src/App.js

```diff
@@ -1,13 +1,13 @@
-import {Router} from "@reach/router";
-import Header from "./components/Header.js";
-import Main from "./components/Main.js";
-import About from "./components/About.js"
-import Work from "./components/Work.js";
-import Residential from "./components/Residential.js";
-import Industrial from "./components/Industrial.js";
-import Contact from "./components/Contact.js";
-import Sitemap from './components/Sitemap.js';
-import Footer from "./components/Footer.js";
+import { Router } from "@reach/router";
+import { Header } from "./components/Header.js";
+import { Main } from "./components/Main.js";
+import { About } from "./components/About.js"
+import { Work } from "./components/Work.js";
+import { Residential } from "./components/Residential.js";
+import { Industrial } from "./components/Industrial.js";
+import { Contact } from "./components/Contact.js";
+import { Sitemap } from './components/Sitemap.js';
+import { Footer } from "./components/Footer.js";
```

Comment:
not important/don't need to change but fyi - you don't need to add the file extensions on import for js (.js)